src/edu/ucsb/cs56/projects/games/states_quiz/QuestionManager.java

```diff
@@ -125,35 +128,56 @@ public void setGameMode(String mode) {
         this.gameMode = mode;
     }
 
+<<<<<<< HEAD
 	/**
 	 * Asks the next question if the question counter is less than 50. Prints
 	 * out the current state.
 	 */
-	public void askNextQuestion() {
-		if (!randStateIndexes.isEmpty()) {
+    public int getRandStateIndex(){
```

Comment:
I'm confused by the name of this function.  Does it really do what it says it does?